src/controllers/search.controller.ts

```diff
@@ -0,0 +1,21 @@
+import { Request, Response } from 'express';
+import * as SearchService from '../services/search.service';
+
+export const search = async (req: Request, res: Response) => {
+  const keyword = req.query.keyword as string;
+  const lng = (req.query.lng as string) || 'en';
+  if (!keyword) {
+    return res.status(400).json({ message: 'Keyword is required' });
+  }
```

Comment:
với chức năng tìm kiếm thì khi ng dùng ko nhập từ khóa sẽ vẫn trả về toàn bộ ds, ko nên trả về lỗi như này e nhé